lib/fluent/plugin/in_tail.rb

```diff
@@ -263,7 +270,9 @@ def expand_paths
           paths += Dir.glob(path).select { |p|
             begin
               is_file = !File.directory?(p)
-              if File.readable?(p) && is_file
+              if (File.readable?(p) ||
```

Comment:
Maybe, one line is more readable. 3 line conditions are hard for me.

Reply:
Fixed in https://github.com/fluent/fluentd/pull/3155/commits/e4a833a5f7514ee7f2c6a36924b92e4456c0f8a2